Lib/typing.py

```diff
@@ -1395,9 +1395,26 @@ def _get_protocol_attrs(cls):
     return attrs
 
 
-def _is_callable_members_only(cls):
+def _is_callable_or_classvar_members_only(cls):
+    attr_names = _get_protocol_attrs(cls)
+    annotations = getattr(cls, '__annotations__', {})
     # PEP 544 prohibits using issubclass() with protocols that have non-method members.
-    return all(callable(getattr(cls, attr, None)) for attr in _get_protocol_attrs(cls))
+    for attr_name in attr_names:
+        attr = getattr(cls, attr_name, None)
+        if callable(attr):
+            continue
+        annotation = annotations.get(attr_name)
+        if getattr(annotation, '__name__', None) == 'ClassVar':
+            continue
+        # String / PEP 563 annotations
+        # Note: If PEP 649 is accepted, we can probably drop this.
+        if isinstance(annotation, str):
+            if (annotation.startswith('ClassVar[')
+               or annotation.startswith('typing.ClassVar[')
+               or annotation.startswith('typing_extensions.ClassVar[')):
+                continue
```

Comment:
1. This is not about PEP 563, it's about forward references, and those could be strings before PEP 563 and will remain so even if PEP 649 is accepted.
2. You can use a single `startswith` and provide a tuple of values to it, see: https://github.com/python-attrs/attrs/pull/367/files
3. There's no precedent of mentioning `typing_extensions` in `typing.py` or CPython docs. In particular, for ClassVar that was added all the way in 3.5.3, I frankly don't think it's worth it.

Reply:
1. Yep, misleading comment there.
2. WOW. I did not know `startswith` supported tuples! Thanks
3. Sure.